### PR TITLE
Update librhsp

### PR DESCRIPTION
**Pull request by LandryNorris**
Update librhsp to [latest](https://github.com/REVrobotics/librhsp/releases/tag/1.0.0)

### CODE REVIEW

**Review by jfabellera — APPROVED** [review]

File: packages/rhsplib/RHSPlib/src/revhub.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    internalHub->serialPort = NULL;
+    RhspModuleInterfaceListInternal* list = (RhspModuleInterfaceListInternal*) internalHub->interfaceList;
+    if (list)
+    {
+        for (int i = 0; i < list->numberOfInterfaces; i++)
+        {
+            free(list->interfaces[i].name);
+        }
+    }
+    free(list);
```
Comment (LandryNorris) [inline packages/rhsplib/RHSPlib/src/revhub.c:67]: Do we want to support a case where a hub can be re-opened after close? If so, I need to nullify the list.

File: packages/rhsplib/src/RevHubWrapper.cc [hunk 2]
```diff
@@ -183,41 +170,41 @@ Napi::Value RevHub::open(const Napi::CallbackInfo &info) {
...
 
-    return Napi::Boolean::New(env, RHSPlib_isOpened(&this->obj));
+    return Napi::Boolean::New(env, rhsp_isOpened(this->obj));
 }
 
 void RevHub::close(const Napi::CallbackInfo &info) {
     Napi::Env env = info.Env();
 
-    RHSPlib_close(&this->obj);
+    rhsp_close(this->obj);
```
[the remark this answers is not in the document]
Comment (NoahAndrews) [reply packages/rhsplib/src/RevHubWrapper.cc:179]: https://github.com/nodejs/node-addon-api/blob/main/doc/object_lifetime_management.md

[the remark this answers is not in the document]
Comment (LandryNorris) [reply packages/rhsplib/src/RevHubWrapper.cc:179]: Alternatively (or additionally), we could throw an exception when a method gets called after close.

File: packages/rhsplib/RHSPlib/src/revhub.c (hunk 1 above)
Comment (NoahAndrews) [reply packages/rhsplib/RHSPlib/src/revhub.c:67]: Nah

File: packages/rhsplib/src/RevHubWrapper.cc (hunk 2 above)
[the remark this answers is not in the document]
Comment (NoahAndrews) [reply packages/rhsplib/src/RevHubWrapper.cc:179]: This function also needs to stop sending keepalives.

**Review by NoahAndrews — APPROVED** [review]

**Pull request merged**